internal/mux/buffer.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2019 The Oto Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mux
+
+import (
+	"bytes"
+	"sync"
+)
+
+// NewConcurrentBuffer returns a ready-to-use ConcurrentBuffer.
+func NewConcurrentBuffer() ConcurrentBuffer {
+	return ConcurrentBuffer{
+		buf: &bytes.Buffer{},
+		m:   &sync.Mutex{},
```

Comment:
Initializing member variables are no longer needed.

Reply:
Going to also remove `NewConcurrentBuffer`.